pandas/tests/test_format.py

```diff
@@ -55,11 +53,7 @@ def has_horizontally_truncated_repr(df):
     except:
         return False
     # Make sure each row has this ... in the same place
-    r = repr(df)
```

Comment:
I would restore this section. You are testing something slightly different here.